schema/cosa/schema_doc.go

```diff
@@ -211,6 +211,7 @@ var generatedSchemaJSON = `{
    "amis",
    "azure",
    "azurestack",
+   "baseos-container",
```

Comment:
Bikeshed: should this be `base-oscontainer` instead for consistency with the existing `oscontainer`?

Or maybe `native-oscontainer` to better highlight the difference between the two.

Reply:
OK renamed to `base-oscontainer`.